src/components/ProgressForm/ProgressForm.module.scss

```diff
@@ -23,6 +23,16 @@ $section-padding-left-right: $base-unit * 25;
     font-size: $base-unit * 4;
 }
 
+.progressHeader_container {
```

Comment:
Hey @Pratiyushkumar, after a discussion with @yesyash, we concluded to use the class name as a banner instead of the header for better understanding and readability. Could we incorporate it here? (I know it's last minute, Sorry for that)

Reply:
fixed